Ginger/GingerCoreNET/BPMN/BPMNXMLSerializer.cs

```diff
@@ -0,0 +1,458 @@
+﻿using NUglify;
+using OctaneStdSDK.Entities.Tasks;
+using OpenQA.Selenium.DevTools.V113.CSS;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+
+#nullable enable
+namespace Amdocs.Ginger.CoreNET.BPMN
+{
+    public sealed class BPMNXMLSerializer
+    {
+        private const string BPMN_XML_PREFIX = "bpmn";
+        private const string BPMN_XML_URI = "http://www.omg.org/spec/BPMN/20100524/MODEL";
+        private const string IG_XML_PREFIX = "ig";
+        private const string IG_XML_URI = "http://insightguard.com/schema/1.0/bpmn";
+
```

Comment:
Please add documentation to function and code wherever possible

Reply:
Added summary to public methods and classes